lib/hydra/derivatives/processors/active_encode.rb

```diff
@@ -0,0 +1,53 @@
+require 'active_encode'
+
+module Hydra::Derivatives::Processors
+  class ActiveEncodeError < StandardError
+    def initialize(status, source_path, errors = [])
+      msg = "ActiveEncode status was \"#{status}\" for #{source_path}"
```

Comment:
There currently isn't any i18n in this gem and I haven't found any examples of non-UI strings using i18n in hydra gems.  Do you think it is still worth doing?